lib/actions/api.js

```diff
@@ -114,7 +114,12 @@ export function updateOtpUrlParams(state, searchId) {
   const { config, currentQuery } = state.otp
   // Get updated OTP params from current query.
   const otpParams = getRoutingParams(config, currentQuery, true)
-  removeUnusedQueryParams(otpParams)
+
+  // Remove unneeded URL params.
+  delete otpParams.ignoreRealtimeUpdates
+  if (otpParams.numItineraries === getDefaultNumItineraries(config)) {
+    delete otpParams.numItineraries
```

Comment:
@miles-grant-ibigroup  Related to https://github.com/opentripplanner/otp-react-redux/pull/1235/files#r1678246983, with this change, `numItineraries` will be not present in the URL by default, so the config setting should still prevail in most cases.

Reply:
This should be fine! Thanks for checking